package.json

```diff
@@ -58,7 +58,7 @@
     "regenerator-runtime": "^0.13.11",
     "sbc-common-components": "3.0.8",
     "uuid": "^9.0.0",
-    "vue": "2.7.14",
+    "vue": "2.7.16",
```

Comment:
Until I updated this, all the unit tests failed (except 3), probably due to a package dependency.

Reply:
That's weird though, I don't see any game breaking package changes here that rely on `2.7.16` 😕

Reply:
Unit tests are passing on my side, that's why I'm confused. Anyway, sounds good.

Reply:
Did you do a `npm i` (or `npm ci`) first?

Reply:
Never mind, I cloned your branch and I see the unit tests failing. Weird though as before, things were looking fine.